src/main/java/organice/ui/FormUiManager.java

```diff
@@ -38,24 +39,25 @@
  */
 public class FormUiManager {
 
-    private static final String COMMAND_EXIT = "/abort";
+    private static final String COMMAND_ABORT = "/abort";
+    private static final String COMMAND_EXIT = "/exit";
```

Comment:
Wouldn't it be quite misleading to have an `/exit` command and another `exit` command? And why do we need to have different commands?

Reply:
cos during form maybe someones name is `exit` LOL

Reply:
Mr.exit

Reply:
Agree